python/GafferSceneUI/InteractiveRenderUI.py

```diff
@@ -229,6 +229,45 @@ def __startPauseClicked( self, button ) :
 	def __stopClicked( self, button ) :
 		self.getPlug().setValue( GafferScene.InteractiveRender.State.Stopped )
 
+##########################################################################
+# UI for the messages plug that presents the render log
+##########################################################################
+
+class _MessagesPlugValueWidget( GafferUI.PlugValueWidget ) :
+
+	def __init__( self, plug, *args, **kwargs) :
```

Comment:
Remove unused `args`. Forward `kw` to `PlugValueWidget.__init__()`.

Reply:
https://github.com/GafferHQ/gaffer/pull/3790/commits/efe1f7546742d13fd6be2668c1b965578758fad2